mayastor-test/test_replica.js

```diff
@@ -445,15 +445,20 @@ describe('replica', function () {
       exec(URING_SUPPORT_CMD, (error) => {
         if (error) {
           self.skip();
-        } else {
-          self.skip();
         }
+        self.skip();
+        // FIXME enable once a fixed Ubuntu kernel 5.4 is released
+        // done();
       });
     });
 
     it('should create a pool with uring io_if', (done) => {
+      const urDisks = [];
+      for (let i = 0; i < disks.length; ++i) {
+        urDisks[i] = 'uring://' + disks[i];
+      }
       client.createPool(
-        { name: POOL, disks: disks, io_if: enums.POOL_IO_URING },
+        { name: POOL, disks: urDisks, io_if: enums.POOL_IO_URING },
```

Comment:
just a nit: functional style code would be more elegant than imperative for loop: `.. disks: disks.map((d) => `uring://${d}`), ...` .

Reply:
Noted, imperative just comes more naturally to me :)

Reply:
Fixed.